### PR TITLE
LB-353: Improve notice while creation of account per GDPR

### PR DESCRIPTION
**Pull request by paramsingh**
# Summary

* This is a
    * Minor / simple change (like a typo)

# Problem

* JIRA ticket (_optional_): [LB-353](https://tickets.metabrainz.org/browse/LB-353) 

# Solution

Improve notice.

### CODE REVIEW

**Comment by mayhem** [comment]
I was hoping to review this in the context of the dialog that reminds users about us processing their data. Can you please close this and wrap the two into one PR?

**Comment by mayhem** [comment]
Looks simple enough, but I wonder about the case where the user clicks on "disagree", gets logged in an taken the account deletion page. What if they don't delete their account and just navigate away? Did they just circumnavigate the GDPR check?

**Comment by paramsingh** [comment]
Umm, I might have messed a rebase up here, let me fix.

File: listenbrainz/webserver/templates/login/login.html [hunk 1]
```diff
@@ -8,16 +8,21 @@ <h2 class="page-title">Sign in</h2>
...
-      </p>
-  </div>
-  <br/>
-  <div class="well" style="max-width: 600px; margin: 0 auto;">
-    <a href="{{ url_for('login.musicbrainz', next=request.args.get('next')) }}"
-       class="btn btn-primary btn-lg btn-block">Sign in with MusicBrainz</a>
-  </div>
+    <p class="text-danger">Important!</p>
+    <p>
+      By creating a ListenBrainz account, you grant the MetaBrainz Foundation permission to process your listen history (not your personal data from your profile) into data dumps we make publicly available under the <a
```
Comment (alastair) [inline listenbrainz/webserver/templates/login/login.html]: "by creating an account" conflicts with "no, take me to the delete page" below. Is it possible to make a distinction between new and returning users?

File: listenbrainz/webserver/views/index.py [hunk 2]
```diff
@@ -47,46 +50,55 @@ def import_data():
 
 
 @index_bp.route("/download")
+@gdpr_acceptance_required
```
[the remark this answers is not in the document]
Comment (paramsingh) [reply listenbrainz/webserver/views/index.py]: I thought we didn't want the user to use ListenBrainz at all until they accepted the terms, that's why I used a different decorator.

[the remark this answers is not in the document]
Comment (paramsingh) [reply listenbrainz/webserver/views/index.py]: Also, alastair's initial suggestion of using `before_request` would redirect for all pages as well.

[the remark this answers is not in the document]
Comment (alastair) [reply listenbrainz/webserver/views/index.py]: Ah, that's an interesting point that I didn't think all the way through. As far as I understand, the major point in which we want to get permission from users is for people who actually have an account - therefore I would only put the check on @Login_required pages (remember that acceptance results in a flag being set on the database, so it only makes sense to happen to logged in users anyway)

[the remark this answers is not in the document]
Comment (alastair) [reply listenbrainz/webserver/views/index.py]: Therefore, if you wanted to do it in @before_request, I'd check first that the user is logged in - it doesn't really matter if they get to the acceptance page from a page like download, it's more important that they get sent to it as soon as they're logged in.

**Comment by paramsingh** [comment]
:tada: :tada: :tada:

**Pull request merged**